src/clientmetricreport/DefaultClientMetricReport.ts

```diff
@@ -372,6 +372,11 @@ export default class DefaultClientMetricReport implements ClientMetricReport {
       media: MediaType.VIDEO,
       dir: Direction.UPSTREAM,
     },
+    videoUpstreamPacketLossPercent: {
```

Comment:
As discussed, should we add these new metrics to the demo?

Reply:
Added in the meeting demo app.